notebooks/304-pytorch-post-training-quantization/README.md

```diff
@@ -0,0 +1,18 @@
+# Optimizing PyTorch models with Neural Network Compression Framework of OpenVINO by 8-bit quantization.
+
+This tutorial demonstrates how to use [NNCF](https://github.com/openvinotoolkit/nncf) 8-bit quantization in post-training mode (without the fine-tuning pipeline) to optimize a 
+[PyTorch](https://pytorch.org/) model for the high-speed inference via [OpenVINO Toolkit](https://docs.openvinotoolkit.org/). 
+For more advanced usage refer to these [examples](https://github.com/openvinotoolkit/nncf/tree/develop/examples).
+
+This notebook is based on 'ImageNet training in PyTorch' [example](https://github.com/pytorch/examples/blob/master/imagenet/main.py).

```

Comment:
I don't like this statement. I'd like every notebook be maximum independent. I propose to remove it or say at the end that "Refer to 'ImageNet training in PyTorch' [example] for 8-bit quantization with the fine-tuning by NNCF."

Reply:
Done